helpers/address_book.py

```diff
@@ -70,7 +70,13 @@ def add_birthday(self, birthday):
         self.birthday = Birthday(birthday)
 
     def add_note(self, title, note):
-        self.notes.append(Note(title, note))
+        is_title = False

```

Comment:
Let's use a dictionary where the keys are titles, and the values are lists of notes. e.x.:

```
self.notes = {}

if title in self.notes:
    self.notes[title].append(note)
else:
    self.notes[title] = [note]
```